components/SplitScreen/SplitForm.js

```diff
@@ -294,23 +343,27 @@ export default () => {
               align-items: center;
             `}
           >
-            {inverted ? (
-              <AmountInput
-                symbol="ANT"
-                color={true}
-                value={inputValueAnj}
-                onChange={() => null}
-              />
-            ) : (
-              <AmountInput
-                symbol="ANJ"
-                color={true}
-                value={inputValueAnj}
-                onChange={() => null}
-              />
-            )}
+            <AmountInput
+              symbol={inverted ? 'ANT' : 'ANJ'}
+              color={true}
+              value={inputValueRecipient}
+              onChange={() => null}
+            />
+            <LabelWithOverlay
```

Comment:
Are we able to control the size of this overlay, to make it a bit more wide?

Reply:
We might be able to due to the [API](https://react-bootstrap.github.io/components/overlays/#overlays-api), but, I believe it may be better to leave the overlay as is, due to mobile constraints.